src/full/Agda/TypeChecking/Free/Old.hs

```diff
@@ -0,0 +1,358 @@
+
```

Comment:
Did you bring back this old file by intention?

Reply:
No, I guess it happened during the merge.